.github/workflows/labeler.yml

```diff
@@ -8,7 +8,23 @@ jobs:
   apply:
     runs-on: ubuntu-latest
     steps:
-    - uses: DataDog/labeler@glob-all
+    - name: Add labels
+      uses: DataDog/labeler@glob-all
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: .github/workflows/config/labeler.yml
+
+  check:
```

Comment:
I think this action should be triggered:
- on PR creation
- on the label event: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#label

With the current setup, the action is only triggered on the PR creation.

Reply:
Better split the jobs into 2 workflows then